CMakeLists.txt

```diff
@@ -148,6 +148,7 @@ ELSE()
   ADD_COMPILE_FLAG("-Wno-unused-parameter")
   ADD_COMPILE_FLAG("-fno-omit-frame-pointer")
   IF(WIN32)
+    ADD_LINK_FLAG("-Wl,--stack,8388608")
```

Comment:
please add a comment for this. i assume it's that unix stack sizes are normally 8MB but we need to add this for mingw?